src/main/java-templates/com/easypost/EasyPost.java

```diff
@@ -33,8 +33,6 @@ public abstract class EasyPost {
     public static final String VERSION = "${project.version}";
     public static final String API_BASE = "https://api.easypost.com/v2";
     public static final String BETA_API_BASE = "https://api.easypost.com/beta";
```

Comment:
Question: Do we want to move these API base strings elsewhere? The client object or the Constant file instead?

Reply:
Jack and I discussed this yesterday. I know in C# we wanted to provide a way for users to override the base URL. Do we still want that here as well?

Reply:
This is currently being used in the Utilities file `singleClassURL` function. Should we allow the user to override the base URL?

Reply:
Oh yeah, yes please. It should be an optional param to the Client object. Remove the `/v2` from the base URL since that's the version and make sure the version is a private prop of the Client so only we can set it

Reply:
This will be the fourth parameter in the EasyPostClient constructor, are we allowing users to override the default `/v2` version?

Reply:
No, I said:

> make sure the version is a private prop of the Client so only we can set it

Reply:
Ok, no optional param should be given in the constructor.